cloudsync/sync/state.py

```diff
@@ -994,7 +998,7 @@ def _storage_update(self, ent: SyncEntry):
     def __len__(self):
         return len(self.get_all())
 
-    def update(self, side, otype, oid, path=None, hash=None, exists=True, prior_oid=None):   # pylint: disable=redefined-builtin, too-many-arguments
+    def update(self, side, otype, oid, path=None, hash=None, exists=True, prior_oid=None, accurate=False):   # pylint: disable=redefined-builtin, too-many-arguments
```

Comment:
does the `update` call in the `EventManager` need to be updated to propagate this field?